Wire-iOS/Sources/UserInterface/Folders/FolderCreationController.swift

```diff
@@ -124,10 +124,10 @@ final class FolderCreationController: UIViewController {
             navigationItem.leftBarButtonItem = navigationController?.closeItem()
         }
 
-        let nextButtonItem = UIBarButtonItem(title: FolderCreationName.Button.create.capitalized,
-                                             style: .plain,
-                                             target: self,
-                                             action: #selector(tryToProceed))
+        let nextButtonItem: UIBarButtonItem = .createNavigationRightBarButtonItem(title: FolderCreationName.Button.create.capitalized,
+                                                                                  systemImage: false,
+                                                                                  target: self,
+                                                                                  action: #selector(tryToProceed))
```

Comment:
Suggestion: this may be a bit more readable

```suggestion
        let nextButtonItem: UIBarButtonItem = .createNavigationRightBarButtonItem(
            title: FolderCreationName.Button.create.capitalized,
            systemImage: false,
            target: self,
            action: #selector(tryToProceed)
       )
```